src/utils.js

```diff
@@ -8,39 +10,77 @@ const getModels = (request) => {
   const noRequestModels = !request.models;
   if (noGetDb && noRequestModels) {
     return notImplemented('`request.getDb` or `request.models` are not defined.'
-                   + 'Be sure to load hapi-sequelize before hapi-sequelize-crud.');
+      + 'Be sure to load hapi-sequelize before hapi-sequelize-crud.');
   }
 
   const { models } = noGetDb ? request : request.getDb();
 
   return models;
 };
 
-export const parseInclude = request => {
+export const parseInclude = async(request) => {
```

Comment:
since nestedInclude may loop the **getModelInstance** function, **parseInclude** is now a promise based function, to call this function needs **await**